doc/man3/SSL_get_builtin_sigalgs.pod

```diff
@@ -0,0 +1,38 @@
+=pod
+
+=head1 NAME
+
+SSL_get_builtin_sigalgs - get list of built-in signature algorithms
+
+=head1 SYNOPSIS
+
+ #include <openssl/tls1.h>
+
+ char * SSL_get_builtin_sigalgs(OSSL_LIB_CTX *libctx);
+
+=head1 DESCRIPTION
+
+Return the colon-separated list of built-in and available TLS signature
+algorithms.
+The string returned must be freed by the user using L<OPENSSL_free(3)>.
+
+
+=head1 NOTES
+
+The string may be empty (strlen==0) if none of the built-in TLS signature
+algorithms can be activated, e.g., if suitable providers are missing.
+
+=head1 RETURN VALUES
+
+A newly allocated string is always returned but it may have strlen == 0.
```

Comment:
There might be an allocation error.

Reply:
True. Possible return value NULL now documented.